playground/src/docs.tsx

```diff
@@ -0,0 +1,63 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
![image](https://github.com/microsoft/qsharp/assets/55811729/5cf51193-9aba-4f0f-a841-f4a3b86eb8c1)
Parts of the returned string are in code-context blocks with the \`\`\` ```This is code.``` \`\`\`. Is there any way to render that to make it stand out more as code blocks, like perhaps giving them a slightly different background or put boards around the code blocks? I assume this is a css thing, but maybe it is more difficult than I realize. If it is too tricky to do, that's okay also.

Reply:
I'll see what I can do, but if it requires customization of the markdown renderer, it's probably not worth it. It gets rendered in <pre> section with the <code> block. But so are other fragments, such as function names.

Reply:
Please take a look. I updated the style.
![image](https://github.com/microsoft/qsharp/assets/60718360/b719236b-9e84-4626-9071-7068e11638bc)